composer.lock

```diff
@@ -0,0 +1,4375 @@
+{
```

Comment:
This `composer.lock` is mostly useful if we install tools like `vimeo/psalm` or similar.

Such tools are very much affected by bugfixes that correct type inference or find new issues, so using a `"locked"` dependency set by default is highly endorsed.

I suggest @frankdejonge to enable @dependabot on the repo (something like a monthly auto-merge schedule) to track problems in upstream dependencies, should there be any: that takes care of this `composer.lock` file too.